packages/front-end/components/Share/Presentation.tsx

```diff
@@ -160,31 +146,50 @@ const Presentation = ({
     }
 
     expSlides.push(
-      // @ts-expect-error TS(2345) If you come across this, please fix it!: Argument of type 'Element' is not assignable to pa... Remove this comment to see the full error message
-      <Slide key={expSlides.length}>
+      <Slide key={expSlides?.length ?? 0}>
         <div className="container-fluid">
-          {/* @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'. */}
-          <Heading className="m-0 pb-0">{e.experiment.name}</Heading>
-          <Text className="text-center m-0 mb-4 p-2" fontSize={21}>
-            {/* @ts-expect-error TS(2532) If you come across this, please fix it!: Object is possibly 'undefined'. */}
-            {e.experiment.hypothesis}
+          <Heading className="m-0 pb-0">
+            {e?.experiment?.name ?? "Experiment"}
+          </Heading>
+          <Text className="text-center m-0 mb-4 p-2 px-5" fontSize={21}>
+            {e?.experiment?.hypothesis
+              ? "Hypothesis: " + e.experiment.hypothesis
+              : ""}
+            {e?.experiment?.metrics && (
+              <>
+                <br />
+                <span style={{ fontSize: "0.95rem" }}>
+                  Primary metrics: {e?.experiment?.metrics.join(", ")}
```

Comment:
These are the metric IDs and therefore are not informative (all you see is `met_***, met_***`). Consider getting the metrics right from the snapshot itself.

Reply:
fixed